addons/website_slides_forum/controllers/main.py

```diff
@@ -6,8 +6,21 @@
 
 
 class WebsiteSlidesForum(WebsiteSlides):
+
+    def _prepare_channel_values(self, **kwargs):
```

Comment:
extend `_slide_channel_prepare_values`.

Reply:
Indeed, the function didn't exist before my morning rebase hell :D